spec.html

```diff
@@ -20382,7 +20382,7 @@
                 </tr>
                 <tr>
                   <td style="border-bottom: 1px solid black">
-                    ResolveExport(exportName, resolveSet, exportStarSet)
+                    ResolveExport(bindingName, resolveSet, exportStarSet)
```

Comment:
The change is not correct. In Step 10.f.ii.2 of [ResolveExport](http://tc39.github.io/ecma262/#sec-resolveexport)
> SameValue(resolution.[[exportName]], starResolution.[[exportName]])

This actually needs to be:
> SameValue(resolution.[[bindingName]], starResolution.[[bindingName]])